CHANGELOG.md

```diff
@@ -1,6 +1,50 @@
 # Changelog
 
-## [v1.43 .0](https://github.com/googleforgames/agones/tree/v1.43.0) (2024-08-27)
+## [v1.44.0](https://github.com/googleforgames/agones/tree/v1.44.0) (2024-10-08)
+
+[Full Changelog](https://github.com/googleforgames/agones/compare/v1.43.0...v1.44.0)
+
+**Implemented enhancements:**
```

Comment:
Do we have any breaking change in this release?

Reply:
No, there are no breaking changes in this release. If I missed anything, please let me know!